tests/installation/scc_registration.pm

```diff
@@ -21,7 +21,10 @@ use registration;
 
 sub run() {
     if (!get_var("HDD_SCC_REGISTERED")) {
-        assert_screen("scc-registration", 100);
+        if (!check_screen 'scc-registration', 210) {
+            record_soft_failure 'bsc#990254';
+            assert_screen('scc-registration', 100);
+        }
```

Comment:
that's not what I am saying. I don't see how a check_screen 300 + assert_screen 10 does make any sense but rather the inverse. I would recommend to just leave the timeout as is, exchange the assert_screen to check_screen plus a assert_screen in case it still fails but after a really long time (not the current suse.com issue but probably something more severe or needle match failed).